sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTranslation.java

```diff
@@ -89,8 +95,8 @@ static class BigQueryIOReadTranslator implements TransformPayloadTranslator<Type
             .addNullableBooleanField("use_legacy_sql")
             .addNullableBooleanField("with_template_compatibility")
             .addNullableByteArrayField("bigquery_services")
-            .addNullableByteArrayField("parse_fn")
-            .addNullableByteArrayField("datum_reader_factory")
+            .addNullableByteArrayField("bigquery_reader_factory")
```

Comment:
This is a complex object to serialize. subject to serialization error if there's changes between versions